Solutions/Menes.Abstractions/Menes/Internal/OpenApiLinkOperationMapper.cs

```diff
@@ -64,7 +64,7 @@ public bool TryGetOperationId(object owner, string relationType, out string oper
         /// <inheritdoc/>
         public bool TryGetOperationId(object owner, string relationType, string context, out string operationId)
         {
-            if (ContentFactory.TryGetContentType(owner, out string ownerContentType) && this.TryGetOperationId(ownerContentType, relationType, context, out operationId))
+            if (ContentFactory.TryGetContentType(owner, out string? ownerContentType) && this.TryGetOperationId(ownerContentType!, relationType, context, out operationId))
```

Comment:
Normally, you shouldn't need the `!` here. The way this pattern is supposed to work is that the compiler understands that if `TryGetContentType` returns `true`, then `ownerContentType` will not in fact be `null`.

But that requires `TryGetContentType`'s second parameter to be annotated with `[NotNullIf(false)]`. If you've been updating the content type stuff as part of this overall work, you should add that annotation, and then you won't need the `!` here (and more generally, anyone using this API won't get annoying spurious nullability warnings, and they also won't need to litter their code with `!`).